src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -136,6 +136,9 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
                     name: PSAnsiProgressFeatureName,
                     description: "Enable lightweight progress bar that leverages ANSI codes for rendering"),
+                new ExperimentalFeature(
+                    name: "PSNativeCommandArgumentPassing",
```

Comment:
Should define the experimental feature name as a const string, see line 24 in this file

Reply:
fixed